src/tools/illink/Directory.Build.props

```diff
@@ -9,30 +10,16 @@
     <NetCoreAppTestsCurrent>net7.0</NetCoreAppTestsCurrent>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(IsReferenceAssembly)' == 'true' ">
-    <!-- Since .NET 5 reference assemblies are always produced -->
-    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
     <ProduceOnlyReferenceAssembly>true</ProduceOnlyReferenceAssembly>
     <!-- Used by Arcade to compute OutputPath, IntermediateOutputPath, etc. early in the import chain. -->
     <OutDirName>$(MSBuildProjectName)/ref</OutDirName>
     <!-- Don't try to publish PDBs for ref assemblies that have none. -->
     <PublishWindowsPdb>false</PublishWindowsPdb>
     <RunApiCompat>false</RunApiCompat>
-  </PropertyGroup>
-
-  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
-  <Import Project="eng/Analyzers.props" />
-  <PropertyGroup>
-    <IsPackable>false</IsPackable>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <RepositoryUrl>https://github.com/dotnet/linker</RepositoryUrl>
+    <UseVSTestRunner>true</UseVSTestRunner>
```

Comment:
Probably because the tests used the VSTest runner. But that property is currently set for reference projects only. I think this was a mistake and should be moved into another propertygroup without a condition.

Reply:
Yes, it should be in a property group without a condition and is needed given that by default arcade will try to run the tests using xunit. By setting this flag we let the VSTestRunner decide which testing infra it will use, in this case, nunit.

Reply:
I think it's worth adding the comment there